rstest_macros/src/render/apply_argumets.rs

```diff
@@ -154,6 +158,11 @@ mod should {
         &["a"],
         "fn f<S: AsRef<str>>(a: impl std::future::Future<Output = S>) {}"
     )]
+    #[case::remove_mut(
+        "fn f(a: u32) {}",
```

Comment:
Should be `fn f(mut a: u32) {}`. Isn't it?

Reply:
Oh yes, sure, good catch :smile: